mlflow/store/artifact/http_artifact_repo.py

```diff
@@ -113,11 +113,19 @@ def delete_artifacts(self, artifact_path=None):
         resp = http_request(self._host_creds, endpoint, "DELETE", stream=True)
         augmented_raise_for_status(resp)
 
+    def _mpu_path(self, base_endpoint, artifact_path):
```

Comment:
nit: could you rename this to a self-explanatory name?

Reply:
```suggestion
    def _constrcut_mpu_uri_and_path(self, base_endpoint, artifact_path):
```

Reply:
sounds good to me!

Reply:
then let's go with this name :)